_data/simple-icons.json

```diff
@@ -6127,6 +6127,11 @@
             "hex": "EA4335",
             "source": "https://fonts.gstatic.com/s/i/productlogos/gmail_2020q4/v8/192px.svg"
         },
+        {
+            "title": "GMC",
+            "hex": "E21836",
+            "source": "https://www.gmc.com"
```

Comment:
Notice on [this page](https://gm.com/copyright-trademark) suggests we cannot add this one.

Reply:
I have done the changes.